schema.sql

```diff
@@ -8,4 +8,4 @@ CREATE TABLE animals (
     weight_kg DECIMAL,
     neutered BOOLEAN,
     escape_attempts INT
-);
+);
```

Comment:
- [ ] According to the project requirements, you should add a new column called `species` with a string type (you are able to choose the varchar limit that you want, varchar limit 255 for example). It will look like this:
```
   ALTER TABLE animals ADD species varchar(255);
```
- Kindly add this 👍🏻